x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -401,6 +402,25 @@ export const createAttachments = async (
   return convertCaseToCamelCase(decodeCaseResponse(response));
 };
 
+export const deleteFileAttachments = async ({
+  caseId,
+  fileIds,
+  signal,
+}: {
+  caseId: string;
+  fileIds: string[];
+  signal: AbortSignal;
+}): Promise<void> => {
+  await KibanaServices.get().http.fetch<CaseResponse>(
```

Comment:
nit: I think we should remove the type (`CaseResponse`) as the endpoint does not return a case response.